### ISSUE TITLE
Unable to update content_type default_always_available property

### ISSUE THREAD

**Issue by raupie** [kaliop-uk/ezmigrationbundle#189]
I would like to modify a content_type without having to delete then rebuild it with the default_always_available property.
```
-
  type: content_type
  mode: update
  identifier: article
  default_always_available: true
```
Does nothing.

**Comment by crevillo** [comment]
I think you miss the match part. It should be something like
```
match:
    identifier: yourctype
````

**Comment by raupie** [comment]
It actually works without the 'match' but I added it anyway. Still nothing. I can see that in the update section for content_type mode: update that the default_always_available is not even an option that can be updated through kaliop. At least that's the way it looks like to me.

**Comment by SalvatorePollaci** [comment]
Guys we really need this option. Any chance of adding it?

**Comment by gggeek** [comment]
This is now in master, will be out most likely next week.
Testing will be appreciated.

**Comment by gggeek** [comment]
Fixed in 5.8.0
